mkdud

```diff
@@ -282,7 +282,14 @@ if($opt_create) {
     @dists = sort keys %d;
 
     for (@dists) {
-      die "unsupported dist \"$_\"\n" if !/^((leap)?\d+\.\d+|tw|sle[sd]\d+)$/;
+      if(!/^((leap)?\d+\.\d+|tw|sle[sd]\d+)$/) {
```

Comment:
Much better error handling, but how about telling the user what is accepted here? At least some little hint?

`mkdud --help` tells you somewhere, but it might be a nice touch to add it here, too (where the information is nearby in the code).